components/Recipe/Recipe.js

```diff
@@ -69,7 +69,13 @@ export default function Recipe({
           onClick={onToggleSelection}
           aria-label={isSelected ? "unselect" : "select"}
```

Comment:
You could be a bit more specific for the aria-label: what to select, what to unselect. That might be a bit more understandable by a screenreader

Reply:
good point.